packages/graph/src/graph2/dataGraph.ts

```diff
@@ -87,6 +88,13 @@ export class DataGraph extends Graph2 {
 
     constructor() {
         super();
+        this
+            .vertexRenderer(Vertex3)
+            .centroidRenderer(CentroidVertex3)
+            .edgeColor("#287EC4" as any)
```

Comment:
Why `as any`?

Reply:
For a while it had been PP as a number
I will fix.